transcribe.py

```diff
@@ -23,6 +23,12 @@
 import audio
 import vis
 
+def find_processed_paths(args):
+	json_paths = [os.path.join(args.output_path, os.path.basename(f)) for f in os.listdir(args.output_path) if f.endswith('.json')]
+	replaced_paths = [p.replace(args.output_path, args.input_path[0]) for p in json_paths]
```

Comment:
Multiple input_paths are currently supported. This will not work correctly in that case